common/src/main/java/gov/cms/ab2d/common/service/PropertiesServiceImpl.java

```diff
@@ -115,4 +117,14 @@ private void logErrorAndThrowException(String propertyKey, Object propertyValue)
         log.error("Incorrect value for {} of {}", propertyKey, propertyValue);
```

Comment:
Since these can be updated by the API you'll want to add some sort of validation in the updateProperties method to ensure that they get added and they have correct values.

Reply:
done